gensim/models/wrappers/fasttext.py

```diff
@@ -340,11 +347,11 @@ def load_vectors(self, file_handle):
             dtype = np.dtype(np.float64)
 
         self.num_original_vectors = num_vectors
-        self.wv.syn0_all = np.fromfile(file_handle, dtype=dtype, count=num_vectors * dim)
-        self.wv.syn0_all = self.wv.syn0_all.reshape((num_vectors, dim))
-        assert self.wv.syn0_all.shape == (self.bucket + len(self.wv.vocab), self.vector_size), \
+        self.wv.syn0_ngrams = np.fromfile(file_handle, dtype=dtype, count=num_vectors * dim)
+        self.wv.syn0_ngrams = self.wv.syn0_ngrams.reshape((num_vectors, dim))
```

Comment:
Fasttext_wrapper code breaks here if I use `min_count=2` with my own data, while the default `min_count=5` is still ok. 

```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-8-08cd9b292aba> in <module>()
----> 1 model.train('/home/rilut/fastText/fasttext', '/datadir/all.csv', model='skipgram', min_count=2)

/home/rilut/anaconda2/lib/python2.7/site-packages/gensim-2.3.0-py2.7-linux-x86_64.egg/gensim/models/wrappers/fasttext.pyc in train(cls, ft_path, corpus_file, output_file, model, size, alpha, window, min_count, word_ngrams, loss, sample, negative, iter, min_n, max_n, sorted_vocab, threads)
    221
    222         output = utils.check_output(args=cmd)
--> 223         model = cls.load_fasttext_format(output_file)
    224         cls.delete_training_files(output_file)
    225         return model

/home/rilut/anaconda2/lib/python2.7/site-packages/gensim-2.3.0-py2.7-linux-x86_64.egg/gensim/models/wrappers/fasttext.pyc in load_fasttext_format(cls, model_file, encoding)
    249             model_file += '.bin'
    250         model.file_name = model_file
--> 251         model.load_binary_data(encoding=encoding)
    252         return model
    253

/home/rilut/anaconda2/lib/python2.7/site-packages/gensim-2.3.0-py2.7-linux-x86_64.egg/gensim/models/wrappers/fasttext.pyc in load_binary_data(self, encoding)
    267             self.load_model_params(f)
    268             self.load_dict(f, encoding=encoding)
--> 269             self.load_vectors(f)
    270
    271     def load_model_params(self, file_handle):

/home/rilut/anaconda2/lib/python2.7/site-packages/gensim-2.3.0-py2.7-linux-x86_64.egg/gensim/models/wrappers/fasttext.pyc in load_vectors(self, file_handle)
    349         self.num_original_vectors = num_vectors
    350         self.wv.syn0_ngrams = np.fromfile(file_handle, dtype=dtype, count=num_vectors * dim)
--> 351         self.wv.syn0_ngrams = self.wv.syn0_ngrams.reshape((num_vectors, dim))
    352         assert self.wv.syn0_ngrams.shape == (self.bucket + len(self.wv.vocab), self.vector_size), \
    353             'mismatch between actual weight matrix shape {} and expected shape {}'.format(

ValueError: cannot reshape array of size 211096795 into shape (2121425,100)```

Reply:
Hey @rilut! Using the FastText wrapper for different values of `min_count` parameter is working fine for me. Could you please share the exact code that is causing the problem in your case?
About `num_vectors` value, the values of `num_vectors` and `dim` are actually read from the files generated from FastText's original C++ code and thus the wrapper code uses the same values at the time of reshaping. 
And apologies for the delayed response. I have been a little occupied recently.